api/page/[id].ts

```diff
@@ -6,6 +6,7 @@ import { fetchBlocks, fetchPageById } from '../../src/notion';
 import { BlockType } from '../../src/types';
 import { getTableData } from '../../src/table';
 
+const QUARTER_IN_SECONDS = 900;
```

Comment:
Le nom n'est pas fou c'est vrai je peux prendre ton `FIFTEEN_MINUTES_IN_SECONDS`

J'ai bien envie de faire du "WET" ici. Comme ca on pourra facilement changer la durée du cache lié à la fonction. Je me sers de la constante comme label pour éviter d'avoir une valeur qui na pas de sens, mais je n'ai pas envie de la mutualiser. Même si on a très envie de le faire, car dans ce cas les valeurs sont les mêmes.

Pour la doc je veux bien rajouter ce lien https://vercel.com/docs/concepts/edge-network/caching#serverless-functions-(lambdas)
qui explique l'usage du header specifique dans ce cas par rapport à vercel

Reply:
OK. Alors si tu veux une valeur potentiellement différente pour les 2 config, je t'invite à créer un enum avec JSDoc comme ceci :
_(ça mutulaisera les commentaires et ça centralisera la config à un même endroit — c'est toujours galère d'aller chercher les configs dans fichiers où il y a l'implémentation)_

```ts
/**
 * Caching with Vercel Edge Network — Serverless Functions (Lambdas)
 * See: https://vercel.com/docs/concepts/edge-network/caching#serverless-functions-(lambdas)
 */
 enum CacheControlMaxAge {
   Page = 900, // 15 seconds
   Table = 900,
 }
 ```
 
 Qu'en dis-tu ?

Reply:
Dans les moins:
- On perd l'unité de la valeur.
- Les label sont le reflet de l'endroit où on a extrait les valeurs.
Dans les plus:
- On a la doc a un seul endroit, quoique encore j'aurais préféré que la doc soit vers la clé du header plutot que la valeur.

Reply:
> On perd l'unité de la valeur

Selon ta proposition, si plus tard on met par exemple 800 secondes on va devoir changer le nom de la constante ...
Avoir un nom qui reflète la valeur correspond plus à des choses qui ne vont jamais changer durant la durée de vie du projet, comme par exemple :
`const SECONDS_IN_ONE_HOUR = 3600;`

Reply:
Alright du coup je rename en `CACHE_CONTROL_MAX_AGE` mes constantes ?

Reply:
Si tu préfères laisser la config dans le fichier, oui.

## Par contre, comme je vois ça :

https://github.com/mobile-club/workspace/pull/5402/files#diff-907cf1ab280a67bbc19f3cb0c082527e1b541b846b28e0ee83036c5f2dcf545d

Je me dis que ça justifier encore + l'utilisation d'un fichier de config qui centralise toutes ces valeurs.

```ts
/**
 * ## Caching with Vercel Edge Network — Serverless Functions (Lambdas)
 * See: https://vercel.com/docs/concepts/edge-network/caching#serverless-functions-(lambdas)
 */
 export enum CacheControlMaxAge {
   Page = 900, // 15 seconds
   Table = 900,
 }
 
/**
 * ## It may take up to 5 days for GoCardless to validate the payment
 * Even after receiving a `Action.Confirmed` event we may receive an
 * `Action.Failed` within 5 days\
 * See: https://developer.gocardless.com/api-reference/#payment-confirmed
  */
 export const GocardlessValidationDelay = 432_000_000; // 5 days in ms
 ```